docs/rules/component-tags-order.md

```diff
@@ -14,27 +14,28 @@ since: v6.1.0
 
 ## :book: Rule Details
 
-This rule warns about the order of the `<script>`, `<template>` & `<style>` tags.
+This rule warns about the order of the `<script>`, `<script setup>`, `<template>` & `<style>` tags.
 
 ## :wrench: Options
 
 ```json
 {
   "vue/component-tags-order": ["error", {
-    "order": [ [ "script", "template" ], "style" ]
+    "order": [ [ "script", "script/setup", "template" ], "style" ]
```

Comment:
Perhaps if it can support CSS selectors (eg `[arrt]`, `[attr=val]`, `:not()`), I think it's easier for users to learn, but I think implementing is difficult.

```js
"vue/component-tags-order": {
  "order": [
     "script:not([setup])",
     "script[setup]",
     "template",
     "style:not([scoped])",
     "style[scoped]",
     "i18n[locale=en]",
     "i18n[locale=ja]",
  ]
}
```

Reply:
Hmm. We may not be able to use esquery.
The esquery attribute selector seems to be very different from the CSS selector.